lib/generators/declare_schema/migration/migrator.rb

```diff
@@ -72,19 +72,30 @@ def constraint_specs
       class Migrator
         class Error < RuntimeError; end
 
-        DEFAULT_CHARSET   = "utf8mb4"
-        DEFAULT_COLLATION = "utf8mb4_bin"
+        DEFAULT_CHARSET               = "utf8mb4"
+        DEFAULT_COLLATION             = "utf8mb4_bin"
+        DEFAULT_TEXT_LIMIT            = 0xffff_ffff
+        DEFAULT_STRING_LIMIT          = nil
+        DEFAULT_NULL                  = false
+        DEFAULT_GENERATE_FOREIGN_KEYS = true
+        DEFAULT_GENERATE_INDEXING     = true
 
         @ignore_models                        = []
         @ignore_tables                        = []
         @before_generating_migration_callback = nil
         @active_record_class                  = ActiveRecord::Base
         @default_charset                      = DEFAULT_CHARSET
         @default_collation                    = DEFAULT_COLLATION
+        @default_text_limit                   = DEFAULT_TEXT_LIMIT
+        @default_string_limit                 = DEFAULT_STRING_LIMIT
+        @default_null                         = DEFAULT_NULL
+        @default_generate_foreign_keys        = DEFAULT_GENERATE_FOREIGN_KEYS
+        @default_generate_indexing            = DEFAULT_GENERATE_INDEXING
```

Comment:
Does anyone use these DEFAULT_ constants outside of here? If not, I'd just get rid of them and put the defaults right there, inline. It will be very clear what they're for.